src/ecstatic/runners/TAJSRunner.py

```diff
@@ -0,0 +1,74 @@
+import os.path
+from typing import List, Dict
+
+from src.ecstatic.models.Level import Level
+from src.ecstatic.models.Option import Option
+from src.ecstatic.runners.CommandLineToolRunner import CommandLineToolRunner
+from src.ecstatic.util.UtilClasses import BenchmarkRecord
+
+
+class TAJSRunner (CommandLineToolRunner):
+    def get_timeout_option(self) -> List[str]:
+        if self.timeout is None:
+            return []
+        else:
+            return f"-time-limit {self.timeout * 60}".split(" ")
+
+    def get_whole_program(self) -> List[str]:
+        # shouldn't be necessary
+        return []
+
+    def get_input_option(self, benchmark_record: BenchmarkRecord) -> List[str]:
+        return f" {benchmark_record.name}".split()
```

Comment:
```suggestion
        return f"{benchmark_record.name}".split()
```

I don't think this makes a difference either way but no reason to have the leading space.